src/components/Trade/hooks/useSwapper/calculateAmounts.ts

```diff
@@ -30,18 +31,25 @@ export const calculateAmounts = ({
   sellAssetUsdRate,
   action,
   selectedCurrencyToUsdRate,
-  tradeFee,
+  sellAssetTradeFeeUsd,
+  buyAssetTradeFeeUsd,
 }: CalculateAmountsArgs): CalculateAmountsReturn => {
   const assetPriceRatio = bnOrZero(buyAssetUsdRate).dividedBy(sellAssetUsdRate)
   const usdAmount = bnOrZero(amount).dividedBy(selectedCurrencyToUsdRate)
   const cryptoSellAmount = toBaseUnit(usdAmount.dividedBy(sellAssetUsdRate), sellAsset.precision)
   const cryptoBuyAmount = toBaseUnit(usdAmount.dividedBy(buyAssetUsdRate), buyAsset.precision)
-  const tradeFeeBaseUnit = toBaseUnit(tradeFee, buyAsset.precision)
+  const sellAssetTradeFeeUsdBaseUnit = toBaseUnit(
```

Comment:
This doesn't refer to a fiat fee anymore
```suggestion
  const sellAssetTradeFeeBaseUnit = toBaseUnit(
```